src/wp-includes/block-supports/layout.php

```diff
@@ -140,6 +140,40 @@ function wp_get_layout_style( $selector, $layout, $has_block_gap_support = false
 	return $style;
 }
 
+/**
+ * Generates the utility classnames for the given blocks layout attributes.
+ * This method was primarily added to reintroduce classnames that were removed
+ * in the 5.9 release (https://github.com/WordPress/gutenberg/issues/38719), rather
+ * than providing an extensive list of all possible layout classes. The plan is to
+ * have the style engine generate a more extensive list of utility classnames which
+ * will then replace this method.
+ *
+ * @param array $block_attributes Array of block attributes.
+ *
+ * @return array Array of CSS classname strings.
```

Comment:
```suggestion
 * @param array $block_attributes Array of block attributes.
 * @return string[] Array of class names.
```

Nit: WP Docs standards don't include a space between params and return.

Reply:
fixed